core/server/middleware/static-theme.js

```diff
@@ -23,7 +29,7 @@ function forwardToExpressStatic(req, res, next) {
 
 function staticTheme() {
     return function blackListStatic(req, res, next) {
```

Comment:
suggestion: rename the function to `blackOrWhitelistStaticFiles` (or something similar)